### PR TITLE
Add "subclass of" property setting

### PR DESCRIPTION
**Pull request by yaronkoren**

### CODE REVIEW

**Comment by yaronkoren** [comment]
@sahajsk21 - if you see this PR (and you approve it), please do a "Squash and merge", if possible, instead of a regular "Merge pull request" - it will look better in the commit history.

File: src/components/subclass.js
Comment (sahajk21) [inline src/components/subclass.js:1]: Contains one more id at line 130

File: src/components/superclass.js
Comment (sahajk21) [inline src/components/superclass.js:1]: Line 129 in this

File: src/components/subclass.js
Comment (yaronkoren) [reply src/components/subclass.js:1]: Thanks for pointing that out - I didn't notice.

**Pull request merged**